### DISCUSSION TITLE
Feature Request: Rename Git Default Branch from "Master" to "Main" for New Projects

### DISCUSSION THREAD

**Discussion by paulsterling** [umbraco/Umbraco.Cloud.Issues#908] [Ideas]
### Issue description

To align with modern Git practices and terminology, I propose renaming the default Git branch from "master" to "main" for all new projects on Umbraco Cloud. We see  an industry-wide shift towards more inclusive language and I think Umbraco Cloud projects should follow the latest Git conventions.

**Comment by timeverts** [comment]
I'd also be keen to see this happen, or at least provide a configuration option in the Cloud Portal for a new project to specify the default branch name. Has there been any further discussion at @UmbracoHQ about this?
